src/Catty/ViewController/BaseViewController/BaseTableViewController.m

```diff
@@ -40,7 +40,7 @@ @interface BaseTableViewController ()
 @property (nonatomic, strong) LoadingView* loadingView;
 @property (nonatomic, strong) UIBarButtonItem *selectAllRowsButtonItem;
 @property (nonatomic, strong) UIBarButtonItem *normalModeRightBarButtonItem;
-@property (nonatomic, strong) ScenePresenterViewController *scenePresenterViewController;
+@property (nonatomic, strong) StagePresenterViewController *scenePresenterViewController;
```

Comment:
again - "stagePresenterViewController" instead of "scenePresenterViewController"